packages/ckb/src/types/spore.ts

```diff
@@ -51,6 +51,21 @@ export interface SporeCreateVirtualTxResult {
   clusterCell: IndexerCell;
 }
 
+export interface BuildAppendingIssuerCellToSporeCreateTx {
```

Comment:
I see the other types here are either suffixed with `Params` or `Result`, only the issuer cell related signatures are exceptions. Are they by design? The exception types: `BuildAppendingIssuerCellToSporeCreateTx`, `AppendIssuerCellToSporeCreate`

Reply:
This is a historical issue. In order to maintain compatibility, it is recommended not to change it.

Reply:
I rename the `BuildAppendingIssuerCellToSporeCreateTx` to `BuildAppendingIssuerCellTxParams` in https://github.com/ckb-cell/rgbpp-sdk/pull/171/commits/18a1c387c6a774028f7e9dab15fcdad0e7cc5bbf.

I suggest keeping the `AppendIssuerCellToSporeCreate` not changed